src/app/components/Notifications/Notifications.contracts.ts

```diff
@@ -24,11 +24,32 @@ export interface NotificationTransactionItemProperties {
 	containmentRef?: any;
 	onVisibilityChange?: (isVisible: boolean) => void;
 	onTransactionClick?: (item?: DTO.ExtendedConfirmedTransactionData) => void;
-	isCompact: boolean;
 }
 
 export interface NotificationsProperties {
 	profile: Contracts.IProfile;
 	onNotificationAction?: (id: string) => void;
 	onTransactionClick?: (item?: DTO.ExtendedConfirmedTransactionData) => void;
 }
+
+export interface NotificationsMigrationItemProperties {
+	// @TODO: assign a proper type for this once defined
+	transaction: any;
+	profile: Contracts.IProfile;
+	// @TBD (also @TODO set transaction type)
+	onClick?: (transaction: any) => void;
```

Comment:
@alfonsobries same here. This can be `DTO.ExtendedConfirmedTransactionData` similar to https://github.com/ArdentHQ/arkvault/pull/286/files#diff-493165b65171712d084660d0e622b82a68beb06d2d20c7886ea9a54b9ee0adffR32